client/src/nv_ingest_client/primitives/tasks/extract.py

```diff
@@ -32,26 +32,41 @@
 ADOBE_CLIENT_SECRET = os.environ.get("ADOBE_CLIENT_SECRET", None)
 
 _DEFAULT_EXTRACTOR_MAP = {
+    # Image files
     "bmp": "image",
-    "csv": "pandas",
-    "docx": "python_docx",
-    "excel": "openpyxl",
-    "html": "txt",
     "jpeg": "image",
     "jpg": "image",
-    "parquet": "pandas",
-    "pdf": "pdfium",
     "png": "image",
-    "pptx": "python_pptx",
-    "text": "txt",
     "tiff": "image",
-    "txt": "txt",
-    "xml": "lxml",
-    "mp3": "audio",
-    "wav": "audio",
+    # Text files
+    "html": "txt",
     "json": "txt",
     "md": "txt",
     "sh": "txt",
+    "text": "txt",
+    "txt": "txt",
+    "log": "txt",
+    "yaml": "txt",
+    "yml": "txt",
+    "ini": "txt",
+    "conf": "txt",
+    "cfg": "txt",
+    "tex": "txt",
+    "rst": "txt",
+    "adoc": "txt",
+    # Data files
+    "csv": "pandas",
+    "parquet": "pandas",
```

Comment:
Should probably discuss the intent with Parquet/excel, since those files have binary/structural data that we may not want to use directly. Maybe dump both of them to CSV first?

@edknv I know there is a follow on task to make all this cleaner and differentiate document types that do and do not need extractions. I think this starts to motivate that a bit more, but also maybe we need to consider some type of ConvertToText stage that acts as a default extractor, and lets us remove structural data from files like Parquet before embedding.